<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="jietushibie">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>截图识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_file">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>540</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_camera">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="jilu">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>546</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>花费</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_meoney">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="off_ve">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>标记违规</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
